playwright/e2e/ui/fixtures/ui.py

```diff
@@ -0,0 +1,42 @@
+import requests
+
+import pytest
+import pytest_asyncio
+
+from playwright.async_api import async_playwright
+
+
+@pytest_asyncio.fixture
+async def chrome_page():
+    async with async_playwright() as playwright:
+        browser_obj = playwright.chromium
+        if browser_obj:
+            ch_browser = await browser_obj.launch(headless=True, slow_mo=1200, timeout=120000)
+            context = await ch_browser.new_context()
+            page = await context.new_page()
+            yield page
+
+            await ch_browser.close()
+
+
+@pytest.fixture
+def abl_approved():
+    """Return list of dictionaries of approved books in ABL json"""
+    abl_url = "https://corgi.ce.openstax.org/api/abl/"
```

Comment:
Interesting... there's another list at https://raw.githubusercontent.com/openstax/content-manager-approved-books/main/approved-book-list.json that Exercises uses